packages/@instana-integration/jenkins/events/executor-exhaustion.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Jenkins Executor Exhaustion",
+  "description": "Detects when Jenkins executors are fully utilized, which may cause build queuing and delays in CI/CD pipelines.",
+  "query": "entity.type:\"jenkins-service\"",
```

Comment:
If this is really needed? This requires collector to add the jenkins-service entity.type always.